homu/init.sls

```diff
@@ -1,38 +1,38 @@
-https://github.com/barosl/homu:
+homu:
   git.latest:
+    - name: https://github.com/barosl/homu
     - rev: 7ff87321e4e60b4059c5c79637b8e1d3386920e8
     - target: /home/servo/homu
     - user: servo
-    - require_in:
-      - pip: install_homu
-
-/home/servo/homu/cfg.toml:
-  file.managed:
-    - source: salt://homu/cfg.toml
-    - template: jinja
-    - user: servo
-    - group: servo
-    - mode: 644
-    - watch_in:
-      - service: homu
-
-/home/servo/homu/_venv:
   virtualenv.managed:
+    - name: /home/servo/homu/_venv
     - venv_bin: virtualenv-3.4
+    - python: python3
```

Comment:
@Manishearth the `venv_bin` option is what tells salt to use virtualenv-3.4, and the `python` option tells virtualenv to install python3 (and pip3) inside the virtualenv.